pkg/trace/agent/agent.go

```diff
@@ -60,6 +60,9 @@ type Agent struct {
 	// ModifySpan will be called on all spans, if non-nil.
 	ModifySpan func(*pb.Span)
 
+	// ModifyTraceFunc will be called on all traces, if non-nil.
+	ModifyTraceFunc func(*[]*pb.Span)
```

Comment:
nit: to be consistent with `ModifySpan` let's name it `ModifyTrace`? or rename `ModifySpan` to `ModifySpanFunc`?

Reply:
Done!